anda/themes/helium-gtk-theme/anda.hcl

```diff
@@ -0,0 +1,6 @@
+project pkg {
+	arches = ["x86_64", "aarch64"]
```

Comment:
why? It's noarch

Reply:
it was marked as x86 only, I assumed this was the way to do it

Reply:
just checked, fluent is the same setup?

Reply:
where does it say x86 only?

Reply:
anda.hcl

Reply:
If this is noarch, both builders will build noarch too?

Reply:
thus there's no reason to have it build on multiple machines

Reply:
i misunderstood then, lemme fix this

Reply:
we may want to document that that's the procedure for noarch